src/sentry/data/samples/android-transaction.json

```diff
@@ -4,7 +4,7 @@
   "release": "io.sentry.samples.android@1.1.0+2",
   "dist": "2",
   "platform": "java",
-  "message": "",
+  "logentry": {},
```

Comment:
Shouldn't the empty ones be removed?

Reply:
I think these are intentional to avoid the code which injects a default message here so I left them